projects/packages/videopress/src/client/block-editor/blocks/video/components/player/index.native.tsx

```diff
@@ -0,0 +1,51 @@
+/**
+ * External dependencies
+ */
+/**
+ * WordPress dependencies
+ */
+import { SandBox } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { View, Text } from 'react-native';
+/**
+ * Internal dependencies
+ */
+import style from '../../style.scss';
+/**
+ * Types
+ */
+import type { PlayerProps } from './types';
+
+type NativePlayerProps = Pick< PlayerProps, 'html' | 'isRequestingEmbedPreview' | 'isSelected' >;
+
+/**
+ * VideoPlayer react component
+ *
+ * @param {object} props - Component props.
+ * @param {string} props.html - HTML markup for the player.
+ * @param {boolean} props.isRequestingEmbedPreview - Whether the preview is being requested.
+ * @param {boolean} props.isSelected - Whether the block is selected.
+ * @returns {object}                     - React component.
+ */
+export default function Player( {
+	html,
+	isRequestingEmbedPreview,
+	isSelected,
+}: NativePlayerProps ) {
+	// Set up style for when the player is loading.
+	const loadingStyle: { height?: number } = {};
+	if ( ! html || isRequestingEmbedPreview ) {
+		loadingStyle.height = 250;
+	}
+
+	return (
+		<View style={ { ...style[ 'wp-block-jetpack-videopress__video-player' ], ...loadingStyle } }>
```

Comment:
In React Native, you can also achieve this by passing an array ([reference](https://reactnative.dev/docs/0.66/style)):
```suggestion
		<View style={ [ style[ 'wp-block-jetpack-videopress__video-player' ], loadingStyle ] }>
```

Reply:
Is this preferred? The documentation first mentions:
`The style prop can be a plain old JavaScript object.`

Reply:
I'd say so, at least I've seen this format in most of the RN files. Although in terms of performance, I don't think there's a real benefit. Some time ago, I recall it was encouraged to use styles created with `StyleSheet.create` and not modify them due to performance, but seems this is no longer the case ([StackOverflow discussion](https://stackoverflow.com/questions/38958888/react-native-what-is-the-benefit-of-using-stylesheet-vs-a-plain-object)).